sale_layout_category_hide_detail/views/sale_portal_templates.xml

```diff
@@ -0,0 +1,61 @@
+                            </div>
+                            <div t-if="layout_category['hide_detail']" class="col-sm-8 col-xs-8 text-right">
+                                <strong>Subtotal: </strong>
+                                <span t-esc="layout_category['hide_detail_line_subtotal']" t-options="{'widget': 'monetary', 'display_currency': order.pricelist_id.currency_id}"/>
+                            </div>
+                        </div>
+                    </t>
+
+                    <t t-foreach="layout_category['lines']" t-as="ol">
+                        <div class="row orders_vertical_align">
+                            <div t-if="not request.env.user._is_public()" class="visible-lg col-lg-1 text-center">
+                                <span t-field="ol.product_image" t-options="{'widget': 'image', 'class': 'img-rounded', 'style':'width: 48px;height:48px'}" />
+                            </div>
+                            <div id='product_name' class="col-lg-4 col-sm-6 col-xs-3">
+                                <span t-esc="ol.product_id.name"/>
+                            </div>
+                            <div t-if="request.env.user._is_public()" class="visible-lg col-lg-1 text-center" /> <!-- Just for alignment -->
+                                <div class="col-xs-3 text-right">
+                                <span t-field="ol.price_unit" t-options='{"widget": "monetary", "display_currency": order.pricelist_id.currency_id}'/>
+                            </div>
+                            <div class="col-xs-2 text-right">
+                                <t t-if="ol._name == 'sale.order.line'">
+                                    <span t-esc="ol.product_uom_qty"/>
+                                </t>
+                                <t t-if="ol._name == 'account.invoice.line'">
+                                    <span t-esc="ol.quantity"/>
+                                </t>
+                            </div>
+                            <div class="col-sm-2 col-xs-4 text-right">
+                                <span t-field="ol.price_subtotal" t-options='{"widget": "monetary", "display_currency": order.pricelist_id.currency_id}'/>
+                            </div>
+                        </div>
+                    </t>
+
+                    <t t-if="(layout_category_size > 1 or page_size > 1) and layout_category['subtotal'] and not layout_category['hide_detail']" groups="sale.group_sale_layout">
+                        <div class="row orders_vertical_align text-right">
+                            <div class="col-sm-12 col-xs-12">
+                                <strong>Subtotal: </strong>
+                                <t t-set="subtotal" t-value="sum(line.price_subtotal for line in layout_category['lines'])"/>
+                                <span t-esc="subtotal" t-options="{'widget': 'monetary', 'display_currency': order.pricelist_id.currency_id}"/>
```

Comment:
Although this seems correctly formatted without bold, it appears bold any way:

![seleccion_015](https://user-images.githubusercontent.com/7165771/49871630-0cf6bd80-fe17-11e8-953b-97ad4a6acb79.png)

It happens in PDF report and in the portal. Can you check why isn't regular instead bold as the other subtotal?